### ISSUE TITLE
Laravel route caching problem

### ISSUE THREAD

**Issue by chilio** [mcamara/laravel-localization#577]
Hi @mcamara , this absolutely great package, and I see no problems in developing, testing with it. All works superb.
But there is really one important problem, when we go into production.
And I mean route caching mechanisms in Laravel here.
So for now, if you try to use `artisan route:cache` all the great stuff that you've done just evaporates.
Any ideas on how to get around this?

**Comment by DCdeBrabander** [comment]
We have the exact same problem here. Is there any solution to this?

**Comment by daniel-de-wit** [comment]
Support for translated route caching is included since [1.3.11]( https://github.com/mcamara/laravel-localization/releases/tag/1.3.11).

Add the `LoadsTranslatedCachedRoutes` trait to your RouteServiceProvider, like so:

```php
class RouteServiceProvider extends ServiceProvider
{
    use \Mcamara\LaravelLocalization\Traits\LoadsTranslatedCachedRoutes;

    ...
}
```

Then use `php artisan route:trans:cache` instead of `php artisan route:cache`.

---------------------
Before 1.3.11 you can use: https://github.com/czim/laravel-localization-route-cache.
